dspace-api/src/main/java/org/dspace/content/dao/impl/EntityTypeDAOImpl.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.content.dao.impl;
+
+import java.sql.SQLException;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Root;
+
+import org.dspace.content.EntityType;
+import org.dspace.content.EntityType_;
+import org.dspace.content.dao.EntityTypeDAO;
+import org.dspace.core.AbstractHibernateDAO;
+import org.dspace.core.Context;
+
+public class EntityTypeDAOImpl extends AbstractHibernateDAO<EntityType> implements EntityTypeDAO {
+
+    public EntityType findByEntityType(Context context, String entityType) throws SQLException {
```

Comment:
please add the override annotation

Reply:
This has been included in https://github.com/DSpace/DSpace/pull/2411